roles/lnls-ans-role-sirius-hla/tasks/setup-Debian.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Update apt cache
+  apt:
+    update_cache: true
+    cache_valid_time: 86400
+  become: true
+  register: apt_update_cache_result
+  until: apt_update_cache_result is success
+  retries: 5
+  delay: 2
+
+- name: Ensure Sirius Apps packages are installed
+  apt:
+    name: "{{ sirius_apps_packages }}"
+    state: "{{ sirius_apps_packages_state }}"
+    install_recommends: "{{ sirius_apps_install_recommends }}"
+  become: true
+  register: apt_packages_result
+  until: apt_packages_result is success
+  retries: 5
+  delay: 2
+
+- name: Ensure Sirius Apps Python packages are installed
+  pip:
+    name: "{{ sirius_apps_pip_packages }}"
+    state: "{{ sirius_apps_pip_packages_state }}"
```

Comment:
Wrong variable name, should be `hla_apps`

Reply:
done